### ISSUE TITLE
Plugin 18.3.0 disabled theme customizations

### ISSUE THREAD

**Issue by rxbracho** [WordPress/gutenberg#61582]
### Description

Two days ago (May 8, 2024), the Gutenberg plugin updated from 18.2.0 to 18.3.0 and it no longer finds my custom color palette or typography changes. The site seems to display properly but the editor only shows the default colors and fonts.

### Step-by-step reproduction instructions

I'm not sure what to say here. I started editing today a new post and the fonts were wrong, and with the wrong color. I then noticed that my color palette had disappeared from the _Site Editor,_ along with other options.

Looking at the blog itself shows the correct fonts, but even when trying to edit older posts that appear properly in the blog cannot be done in a true WYSISWG fashion, the fonts and other things use the default colors of the theme, for instance.

Disabling the plugin "fixed" the problem, but the default Gutenber editor available in 6.5.3 has many problems, especially in mobile devices. I don't know how to revert to 18.2.0 but I know it was working a few days ago.

### Screenshots, screen recording, code snippet

![Gutenberg_18 3 0_Disabled 1](https://github.com/WordPress/gutenberg/assets/47257701/a9c48bd7-612c-4bcf-9ae1-f65a98002dbf)
![Gutenberg_18 3 0_Enabled 1](https://github.com/WordPress/gutenberg/assets/47257701/136fc4d8-599f-4e50-80dc-d4f976475090)

### Environment info

WordPress 6.5.3
Theme "Twenty twenty-three"
Guttenberg 18.3.0 shows the problem, it disappears when deactivated.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

No

**Comment by rxbracho** [comment]
**UPDATE:** I was able to revert to 18.2.0 and the problem went away, so it is definitely related to 18.3.0

I disabled automatic updates for now, I will be monitoring to see if the bug is fixed in a future release.

**Comment by goodpenguins** [comment]
The same.
When the Gutenberg 18.3.0 is activated, this style is applied in the post editor:

```
/wp-includes/css/dist/block-library/reset.css?ver=6.5.3

:where(.editor-styles-wrapper) {
     background: #fff;
     color: initial;
     font-family: serif;
     font-size: medium;
     line-height: normal;
```

When deactivated it's overridden by the theme.json settings.

**Comment by Mamaduka** [comment]
It's probably the same report as #61565.

@rxbracho, do you have any meta boxes active in the editor, like it's reported in the other issue?

**Comment by rxbracho** [comment]
@Mamaduka I do _not_ have "Custom Fields" activated in Preferences, if that is what you mean. Else, I don't know what you mean by "meta boxes active". I don't do much customizing, I use the vanilla twenty twenty-three with the "marigold" style, but I changed the color of my font, and a few other cosmetic things. To do that I built a color palette that disappeared with 18.3.0 as well.

I apologize for the delayed response, I didn't get a notification.

**Comment by Mamaduka** [comment]
@rxbracho, no problem at all. I'm going to re-open the issue. There's a plan to release a minor plugin update with the recent fix.

If that's not a problem, I would appreciate feedback on whether it fixed the issue for you because I couldn't reproduce it without the "Custom Fields" active.

Thanks again for reporting the bug!

**Comment by rxbracho** [comment]
I have linked my wordpress.org and my github accounts as requested in #61602

**Comment by djcowan** [comment]
``` :where(.editor-styles-wrapper) ```
Happy to find this issue is being looked at; the reset is effecting typographic settings on page/post editor.
`add_editor_style( ) `  doesn't appear to support a priorty and after_setup hook has no effect in overriding the reset.css - which is amusingly ironic given the reset css caution in the [documentation](https://developer.wordpress.org/reference/functions/add_editor_style/#:~:text=wrapper%20%7B%20color%3A%20red%20%7D-,%23%20CSS%20Resets,-If%20you%E2%80%99re%20using)

**Comment by Mamaduka** [comment]
@rxbracho, did you get a chance to test the fix with Gutenberg 18.4 or 18.5?

**Comment by rxbracho** [comment]
> @rxbracho, did you get a chance to test the fix with Gutenberg 18.4 or 18.5?

I apologize, I didn't even realize that new versions had been released.  The system updated me to 18.4.1 just now and the problem appears fixed.  Many thanks to everyone involved.

**Comment by Mamaduka** [comment]
Thanks for the update, @rxbracho!
